docs/editor/integrated-terminal.md

```diff
@@ -19,50 +19,62 @@ To open the terminal:
 
 ![Terminal](images/integrated-terminal/integrated-terminal.png)
 
-> **Note:** You can still open an external terminal with the `kb(workbench.action.terminal.openNativeConsole)` keyboard shortcut if you prefer to work outside VS Code.
+> **Note:** Open an external terminal with the `kb(workbench.action.terminal.openNativeConsole)` keyboard shortcut if you prefer to work outside VS Code.
 
-## Managing multiple terminals
+## Managing terminals
 
-You can create multiple terminals open to different locations and easily navigate between them. Terminal instances can be added by clicking the plus icon on the top-right of the **TERMINAL** panel or by triggering the `kb(workbench.action.terminal.new)` command. This action creates another entry in the dropdown list that can be used to switch between them.
+The terminal tabs view is to the right of the terminals. Each terminal has an entry with its name, icon, color, and group decoration (if any).
 
-![Multiple Terminals](images/integrated-terminal/terminal-multiple-instances.png)
+![Terminal tabs](images/integrated-terminal/tabs.png)
+> **Tip:** Change the tabs location using the `integrated.terminal.tabs.location` setting.
+
+Terminal instances can be added by clicking the plus icon on the top-right of the **TERMINAL** panel, selecting a profile from the new terminal dropdown, or by triggering the `kb(workbench.action.terminal.new)` command. This action creates another entry in the tab list associated with that terminal.
+
+Remove terminal instances by hovering a tab and pressing the trash can button, selecting a tab item and pressing `kbstyle(delete)`, by triggering the `kb(workbench.action.terminal.kill)` command, or via the right click context menu.
+
+Navigate between terminal groups using focus next `kbstyle(workbench.action.terminal.focusNext)` and focus previous `kbstyle(workbench.action.terminal.focusPrevious)`.
+
+Icons may appear to the right of the terminal title on the tab label when the terminal's status change. Statuses include disconnected, in need of relaunch, and bell. Hover the icon to read status information and to take action.
 
-Remove terminal instances by pressing the trash can button.
+### Grouping
 
->**Tip:** If you use multiple terminals extensively, you can add key bindings for the `focusNext`, `focusPrevious` and `kill` commands outlined in the [Key Bindings section](/docs/editor/integrated-terminal.md#terminal-keybindings) to allow navigation between them using only the keyboard.
+Split the terminal by:
+- triggering the `kb(workbench.action.terminal.split)` command
+- right clicking the context menu and selecting the `Split...` submenu option
+- `kbstyle(alt)` clicking a tab, the plus button, or the the single tab on the terminal panel
```

Comment:
Missing clicking the inline/panel split button which will probably be the main way. Let's move the command to the bottom as it's least important.